tests/layer_tests/tensorflow_tests/test_tf_Pad.py

```diff
@@ -75,9 +80,12 @@ def create_pad_complex_net(self, input_shape, pads_values):
         return tf_net, None
 
     test_data_basic = [
-        dict(input_shape=[1, 50], pads_values=[[0, 1], [2, 3]]),
-        dict(input_shape=[2, 20, 10], pads_values=[[0, 1], [2, 3], [4, 0]]),
-        dict(input_shape=[1, 5, 10, 3], pads_values=[[1, 1], [0, 0], [4, 0], [1, 1]]),
+        dict(input_shape=[1, 50], pads_values=[[0, 1], [2, 3]], const_value=None, pad_op='Pad'),
+        dict(input_shape=[2, 20, 10], pads_values=[[0, 1], [2, 3], [4, 0]], const_value=None, pad_op='Pad'),
+        dict(input_shape=[1, 5, 10, 3], pads_values=[[1, 1], [0, 0], [4, 0], [1, 1]], const_value=None, pad_op='Pad'),
+        dict(input_shape=[1, 50], pads_values=[[0, 1], [2, 3]], const_value=(1.0, 0.0), pad_op='PadV2'),
+        dict(input_shape=[2, 20, 10], pads_values=[[0, 1], [2, 3], [4, 0]], const_value=(0.0, 1.0), pad_op='PadV2'),
+        dict(input_shape=[1, 5, 10, 3], pads_values=[[1, 1], [0, 0], [4, 0], [1, 1]], const_value=(1.0, 2.0), pad_op='PadV2'),
     ]
 
     @pytest.mark.parametrize("params", test_data_basic)
```

Comment:
please revert these changes in test and just create separate test suite class `TestComplexPadV2`